artman/tasks/publish/local.py

```diff
@@ -73,7 +73,7 @@ def execute(self, gapic_code_dir, git_repo, local_paths,
         dests = []
 
         # Determine where the code belongs and stage it there.
-        for path in git_repo.get('paths', ['.']):
```

Comment:
I think that an error is a good idea. We should generate an error here, but probably also we could check for this error during pipeline construction, WDYT?